kubernetes/deployment-production.tmpl

```diff
@@ -38,6 +37,10 @@ spec:
             limits:
               memory: "2200Mi"
               cpu: "500m"
+          livenessProbe:
+            httpGet:
+              path: /
```

Comment:
i'm not 100% sure the app returns a 200 response on this path...worth asking dev team for this.